articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -141,69 +141,69 @@ The thumbprint of the certificate(s) is also required. A guide on how to find th
 
-   - **Custom Application Endpoints** - Custom endpoints allow for connections to applications running on this cluster. Enter endpoints separated by comma: 80,8080,8081.
-
-   - **Custom ports to open in the Network Security Group** - Custom ports to allow for the Network Security Group. Enter ports separated by comma: 3389,80,8080,8081.
+   - **Custom Application Endpoints** - Custom endpoints allow for connections to applications running on this cluster. Enter endpoints separated by commas, for example, 80,8080,8081.
 
+   - **Custom ports to open in the Network Security Group** - Custom ports to allow for the Network Security Group. Enter ports separated by comma, for example, 3389,80,8080,8081.
+   
    ![Create Service Fabric cluster > Network Settings](images/azs-browser-create-sf-networking.png) 
 
-6. In the **Node Configuration** blade, enter the following information and click **OK**:
+6. In the *Node Configuration* blade, enter the following information and click **OK**:
 
    - **Image SKU** - The image template that each node is built from.
 
    - **VM Size** - The size of each node.
 
    ![Create Service Fabric cluster > Node Configuration](images/azs-browser-create-sf-node.png)
 
-7. In the **Security** blade, enter the following information and click **OK**:
+7. In the *Security* blade, enter the following information and click **OK**:
 
-   - **Source Key Vault** - The resource ID of the key vault that you gathered in the previous section. This should be in the format of /subscriptions/[subscriptionid]/resourceGropus/[resourceGroupName]/providers/Microsoft.KeyVault/vaults/[vaultName].
+   - **Source Key Vault** - The resource ID of the key vault that you gathered in the previous section. This should be in the format of /subscriptions/<subscriptionid>/resourceGropus/<resourceGroupName>/providers/Microsoft.KeyVault/vaults/<vaultName>.
 
-   - **Cluster Certificate URL** - This refers to the Secret Identifier of the cluster certificate which you gathered in the previous section. It should be in the format of https<!---->://[vaultEndpoint]/secrets/[secretName]/[secretVersion].
+   - **Cluster Certificate URL** - The Secret Identifier of the cluster certificate that you gathered in the previous section. It should be in the format of https<!---->://<vaultEndpoint>/secrets/<secretName>/<secretVersion>.
 
    - **Cluster Certificate thumbprint** - Cluster certificate thumbprint that you gathered earlier. For example, 1742635FBCC5F9A442582516A7292523686DE3D7.
 
-   - **Server Certificate URL** - This refers to the Secret Identifier of the server certificate which you gathered in the previous section. It should be in the format of https<!---->://[vaultEndpoint]/secrets/[secretName]/[secretVersion].
+   - **Server Certificate URL** - The Secret Identifier of the cluster certificate that you gathered in the previous section. It should be in the format of https<!---->://<vaultEndpoint>/secrets/<secretName>/<secretVersion>.

```

Comment:
Escape the angled brackets so they render correctly

Reply:
FIxed